docs/05-technical-reference/00-custom-resources/telemetry-03-tracepipeline.md

```diff
@@ -46,41 +46,41 @@ For details, see the [TracePipeline specification file](https://github.com/kyma-
 <!-- TracePipeline v1alpha1 telemetry.kyma-project.io -->
 | Parameter         | Description                                   |
 | ---------------------------------------- | ---------|
-| **spec.output** | Configures the trace receiver of a TracePipeline. |
-| **spec.output.otlp** | Defines an output using the OpenTelmetry protocol. |
+| **spec.output** | Defines a destination for shipping trace data. Only one can be defined per pipeline. |
+| **spec.output.otlp** | Configures the underlying Otel Collector with an [OTLP exporter](https://github.com/open-telemetry/opentelemetry-collector/blob/main/exporter/otlpexporter/README.md). If you switch `protocol`to `http`, an [OTLP HTTP exporter](https://github.com/open-telemetry/opentelemetry-collector/tree/main/exporter/otlphttpexporter) is used. |
 | **spec.output.otlp.authentication** | Defines authentication options for the OTLP output |
 | **spec.output.otlp.authentication.basic** | Contains credentials for HTTP basic auth |
```

Comment:
I see that you took the existing docu from CRD instead of docu. I think the one in the docu is the better choice, also the sub-elements in basic.password.value have something in the docu